app/store/keys/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './keys.actions';
+export * from './keys.reducer';
```

Comment:
Hm, are you explicitly adding `keys` to make things more searchable or clear from a DX standpoint? I can see why that's better than having 5 `actions.ts` files.

Reply:
This just makes it much easier to scan through visually / fuzzy search imo. Favour explicit naming wrt file names.